config/locales/en.yml

```diff
@@ -6,14 +6,41 @@ en:
         institution_name_full: Full institution name
   application:
     tagline: The next-generation repository solution
+  blacklight:
```

Comment:
remove line 9-29 for this and all other languages

Reply:
lines 9-29 starting with blacklight: has been deleted.